Sources/Protocols/Keyboardable.swift

```diff
@@ -9,26 +9,36 @@
 import Foundation
 import UIKit
 
-public protocol Keyboardable {
+protocol Keyboardable: class {
```

Comment:
Please put back the public accessors, otherwise the Keyboardable will not be available throughout CocoaPods framework